src/Participant.hpp

```diff
@@ -193,23 +197,22 @@ class Participant
             ::fastdds::dds::Topic* topic,
             ::fastdds::dds::DomainEntity* entity);
 
-private:
+protected:
 
-    /**
-     * @brief Create a *Fast DDS DomainParticipant* using a certain profile.
-     *
-     * @note This method is a workaround due to `v2.0.X` versions of *Fast DDS* not including
-     *        this method inside the *DomainParticipantFactory* class.
-     *
-     * @param[in] profile_name The XML profile name for the participant.
-     *
-     * @returns A correctly initialized DomainParticipant.
-     *
-     * @throws DDSMiddlewareException if some error occurs during the creation process.
-     */
-    ::fastdds::dds::DomainParticipant* create_participant_with_profile(
-        const std::string& profile_name);
+    //! Get Participant QoS using config file
+    eprosima::fastdds::dds::DomainParticipantQos get_participant_qos(
+            const YAML::Node& config);
+
+    //! Get Integration Service Participant default Qos
```

Comment:
Comments should follow same doxygen style as the whole file does

Reply:
:+1: